### PR TITLE
Fetch news info if `url` query string param is defined

### LINKED ISSUE #1588: News Search – Fetched text formatted incorrectly
When the article info is pulled from News Search, the text that is automatically fetched from the article has an issue where the Markdown lines within a paragraph are split (similarly to how the lines are displayed in the article), manually resolving this by deleting each new line would take a lot of time.

image

Current workaround: use `Fetch info` button in the Submitter's form to re-fetch the text

### PR DESCRIPTION
**Pull request by pdcp1**
This Closes #1588

### CODE REVIEW

File: site/gatsby-site/cypress/e2e/integration/submit.cy.js [hunk 1]
```diff
@@ -1324,4 +1324,18 @@ describe('The Submit form', () => {
 
     cy.contains('Please review. Some data is missing.').should('not.exist');
   });
+
+  it.only('Should fetch the news if the url param is in the querystring', () => {
```
Comment (cesarvarela) [inline site/gatsby-site/cypress/e2e/integration/submit.cy.js:1328]: leftover `only`

File: site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js [hunk 2]
```diff
@@ -142,6 +144,12 @@ const FormDetails = ({
     }
   }, [submissionFailed, submissionComplete]);
 
+  useEffect(() => {
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js:147]: Does this trigger a fetch if you change the URL manually?

**Review by cesarvarela — CHANGES_REQUESTED** [review]

File: site/gatsby-site/cypress/e2e/integration/submit.cy.js (hunk 1 above)
Comment (pdcp1) [reply site/gatsby-site/cypress/e2e/integration/submit.cy.js]: Fixed

File: site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js (hunk 2 above)
Comment (pdcp1) [reply site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js:147]: Fixed

**Review by cesarvarela — APPROVED** [review]

**Pull request merged**